src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -1243,6 +1257,10 @@ export class SettingObjectRenderer extends AbstractSettingObjectRenderer impleme
 		});
 
 		template.context = dataElement;
+		template.onChange = (v: Record<string, unknown> | undefined) => {
+			renderArrayValidations(dataElement, template, v, false);
```

Comment:
I don't think we should overwrite onChange here

Reply:
I'll create a new template function